### PR TITLE
Counter dispute form controller and submit

### PR DESCRIPTION
**Pull request by jakemer**
Links
-----

Developer considerations
--------------

- On any task
  - [ ] Previous unit and e2e tests are passing
  - [ ] Perform dev QA on Chrome, Firefox, and Safari
- On new features
  - [ ] Add unit tests
  - [ ] Add e2e tests
  - [ ] Add documentation
  - [ ] Does the component have exportedparts for styling? If so, add unit a unit test for each exportedpart
- On bugs
  - [ ] Add unit tests to prevent the bug from happening again

Developer QA steps
--------------------

### CODE REVIEW

File: packages/webcomponents/src/components/dispute-management/dispute-notification.tsx [hunk 1]
```diff
@@ -14,48 +22,74 @@ export class DisputeNotification {
...
-            <span class="fw-bold d-table-cell pe-4">
-              Amount
-            </span>
-            <span class="flex-1 d-table-cell text-wrap">
-              $50.00
-            </span>
+      <div>
+        {this.isLoading && (
+          <div style={{ marginBottom: '8px' }}>
+            <Skeleton height={'28px'} width={'70%'} styles={{ marginBottom: '16px' }} />
```
Comment (jakemer) [inline packages/webcomponents/src/components/dispute-management/dispute-notification.tsx:34]: This doesn't seem to be working after resolving merge conflicts with the release branch

File: packages/webcomponents/src/components/dispute-management/dispute-response/dispute-response-core.tsx [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+    onSuccess: (disputeResponse: any) => void,
+    onError: (disputeResponse: any) => void
+  }) => Promise<IApiResponse<any>>;
+  @Prop() disputeResponse: any = {};
+
+  @State() isLoading: boolean = false;
+  @State() documentList = [];
+  @State() currentStep = 0;
+  @State() currentStepComponentRef: any;
+  @State() formController: FormController;
```
Comment (jakemer) [inline packages/webcomponents/src/components/dispute-management/dispute-response/dispute-response-core.tsx:35]: Is this still needed? Each step has a formController...

File: packages/webcomponents/src/components/dispute-management/dispute-response/dispute-response-core.tsx [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+      await this.handleSubmit(formData, documentList);
+      this.clickEvent.emit({ name: DisputeManagementClickEvents.previousStep });
+      this.currentStep--;
+    });
+  }
+
+  private onNext = async () => {
+    await this.currentStepComponentRef.validateAndSubmit(async (formData, documentList) => {
+      await this.handleSubmit(formData, documentList);
+      this.clickEvent.emit({ name: DisputeManagementClickEvents.nextStep });
```
Comment (jakemer) [inline packages/webcomponents/src/components/dispute-management/dispute-response/dispute-response-core.tsx:172]: Make sure this is consistent with payment provisioning form

File: packages/webcomponents/src/components/dispute-management/dispute-response/dispute-response-form-types.ts [hunk 4]
```diff
@@ -0,0 +1,22 @@
+export interface DisputeResponseFormSubmitEvent {
+  data?: any;
+  metadata?: any;
+}
+
+export interface DisputeResponseFormStepCompletedEvent {
```
Comment (jakemer) [inline packages/webcomponents/src/components/dispute-management/dispute-response/dispute-response-form-types.ts:6]: Revisit this for consistency with payment provisioning form

File: packages/webcomponents/src/components/dispute-management/dispute-response/dispute-response.tsx [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+    onSuccess: (disputeResponse: any) => void,
+    onError: (disputeResponse: any) => void
+  }) => Promise<IApiResponse<any>>;
+  @State() errorMessage: string = null;
+
+  @Prop() authToken: string;
+  @Prop() disputeId: string;
+  @Prop() disputeResponse: DisputeResponse;
+  @Prop() apiOrigin?: string = config.proxyApiOrigin;
+
```
Comment (jakemer) [inline packages/webcomponents/src/components/dispute-management/dispute-response/dispute-response.tsx:38]: Add submitted event?

File: packages/webcomponents/src/components/dispute-management/dispute-response/additional-statement.tsx [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+              multiple={true}
+              helpText="Upload any additional pieces of evidence that have not already been provided."
+              onChange={this.handleFileSelection}
+            />
+          </div>
+          <div class="col-12">
+            <form-control-checkbox
+              name="forfeit"
+              label="I understand that this information can only be submitted once and have provided as much evidence as possible to support my claim."
+              inputHandler={(name, forfeit) => {
```
Comment (jakemer) [inline packages/webcomponents/src/components/dispute-management/dispute-response/additional-statement.tsx:75]: 2 issues with this...
- Not sure if this should really be tied to forfeit, forfeit should probably just be set on POST (`forfeit = false` in this case)
- We do need an acknowledgement mechanism, but it shouldn't validate on "back"

**Review by colganmeanor — APPROVED** [review]
CR and QA approved

Just needs messages to be written for the dispute notification component! 

Great work @jakemer

**Review by TiagoFuelber — APPROVED** [review]
Approved

**Pull request merged**